Update GOVERNANCE.md

CortNick opened the pull request:
Add Kyverno Steering and Oversight Committee (KSOC) description to our project Governance.

CortNick committed:
Update GOVERNANCE.md

Signed-off-by: Cortney <cortney.nickerson@nirmata.com>

JimBugwadia reviewed (commented):
@CortNick - can we move the new section to KSOC.md and add a link to GOVERNANCE.md.

JimBugwadia reviewed (changes requested):
@CortNick - can we move the new section to KSOC.md and add a link to GOVERNANCE.md.

CortNick committed:
Update GOVERNANCE.md

Signed-off-by: Cortney <cortney.nickerson@nirmata.com>

JimBugwadia committed:
Merge branch 'main' into main

JimBugwadia committed:
Merge branch 'main' into main

CortNick committed:
Create KYVERNO_STEERING_OVERSIGHT_COMMITTEE.md

Add KYVERNO_STEERING_OVERSIGHT_COMMITTEE.md with Steering & Oversight Committee bylaws

Signed-off-by: Cortney <cortney.nickerson@nirmata.com>

CortNick committed:
Update GOVERNANCE.md

Move KSOC content out of GOVERNANCE.md and link to KYVERNO_STEERING_OVERSIGHT_COMMITTEE.md


Signed-off-by: Cortney <cortney.nickerson@nirmata.com>

CortNick commented:
@JimBugwadia I have created the KYVERNO_STEERING_OVERSIGHT_COMMITTEE.md and adjusted the GOVERNANCE.md to link to and reflect these changes as requested. Please review and let me know if there is anything else that needs adjusted.

joshgav commented on KYVERNO_STEERING_OVERSIGHT_COMMITTEE:
```diff
@@ -0,0 +1,243 @@
+
+#### 5.2 Termination / Exit
+
+-   A member’s term ends after the two-year period (unless renewed). At least 2 months before end-of-term, the liaison will ask if the member wishes to renew (and maintainers will evaluate).  
```
On line 67 it says the membership term is 6 months and here it says 2 years.

JimBugwadia committed:
Merge branch 'main' into main

CortNick committed:
Update KYVERNO_STEERING_OVERSIGHT_COMMITTEE

I've updated this file to correctly reflect the amount of time each committee member is permitted maintain their seat on.

Signed-off-by: Cortney <cortney.nickerson@nirmata.com>

CortNick committed:
Update KYVERNO_STEERING_OVERSIGHT_COMMITTEE

I have updated this to ensure it is aligned with the extended description in section 5.3

Signed-off-by: Cortney <cortney.nickerson@nirmata.com>

joshgav reviewed (commented):
@CortNick I follow the term length definition now, thanks! I made a couple suggestions to clarify, PTAL.

It's a good idea to revisit and renew memberships every 6 months, but TBH I'm a bit surprised you want to limit how long someone can serve, you may find you want to keep impactful folks around longer. Perhaps drop the 2y limit for now?

I also added a note about specific criteria for changing this doc (that way you can add a term limit later!).

The maintainers should probably approve this initial version too? You could note that at the end - "This document will become effective upon simple majority vote of current maintainers."

CortNick committed:
Merge branch 'kyverno:main' into main

CortNick committed:
Merge branch 'kyverno:main' into main

CortNick committed:
Add AI usage policy for contributors

- Define guidelines for AI-assisted contributions
- Set limits on open PRs and disclosure requirements
- Establish blog post and licensing standards
- Create framework for sustainable AI tool usage in project

Signed-off-by: Cortney <cortney.nickerson@nirmata.com>

CortNick committed:
Merge pull request #2 from CortNick/CortNick-patch-2

Add AI_USAGE_POLICY.md for sustainable contributions

CortNick committed:
Update membership terms and renewal process

Clarified membership renewal process and term details in the committee charter according to governance reviewer comments.

Signed-off-by: Cortney <cortney.nickerson@nirmata.com>

CortNick committed:
Revise committee member term and renewal policies

Updated membership term details for committee members, allowing unlimited consecutive terms and clarifying renewal process.

Signed-off-by: Cortney <cortney.nickerson@nirmata.com>

CortNick reviewed (commented):
@joshgav I didn't realize there was a comment left unresolved. I have reviewed all of this once more and hopefully everything has been addressed this time around. Really appreciate your help and insights about the serving time of each KSOC member. I adjusted accordingly as you made very valid points! Thank you for the time you spent on this and look forward to further comments and review.

CortNick commented on KYVERNO_STEERING_OVERSIGHT_COMMITTEE:
```diff
@@ -0,0 +1,243 @@
+
+#### 5.2 Termination / Exit
+
+-   A member’s term lasts for a six (6) month period and can be renewed for consecutive six (6) month terms for a total length of two (2) years. At least 1 month prior to the end of each six (6) month term, the liaison will ask if the member wishes to renew (and maintainers will evaluate). If the member has reached the maximum permitted renewal time and has completed two (2) years in their post,  ...
```
Done! @joshgav please have a look

CortNick commented on KYVERNO_STEERING_OVERSIGHT_COMMITTEE:
```diff
@@ -0,0 +1,243 @@
+-   When a membership vacancy arises (end-term, resignation, removal), the maintainers will select a replacement from the short-list (or extend new invitation), giving priority to ensuring representation balance (organisation type, region, user vs ecosystem).  
+      
+    
+-   The new member will serve a full two-year term (or the remainder of the outgoing member’s term, per policy) and then be eligible for renewal under the same rules.  
```
DONE! @joshgav please have a look

CortNick commented on KYVERNO_STEERING_OVERSIGHT_COMMITTEE:
```diff
@@ -0,0 +1,243 @@
+
+### Amendment & Review
+
+-   This Committee charter may be reviewed annually (by the maintainers + Committee) and amended by the maintainers in consultation with the Committee.  
```
DONE! @joshgav please have a look

CortNick committed:
Merge branch 'main' into main

Signed-off-by: Cortney <cortney.nickerson@nirmata.com>

joshgav commented:
This is great @CortNick, LGTM!
